### DISCUSSION TITLE
clasp pull behaviour changed?

### DISCUSSION THREAD

**Discussion by PreetM07** [google/clasp#1010] [Q&A]
Hello Team,

Thanks for building clasp.
I was working with a GAS project recently and clasp pull was working beautifully.
I am only using clasp pull because clasp push is reordering the files in the GAS editor and that's something I don't want as of now...
So I have studio code set to the directory of the project, and when I used to do clasp pull, the files were used to reflect live from the GAS changes.
But for 3-4 days, I am not seeing my local repo getting updated even if I do clasp pull...

any idea?

**Comment by Frick-David** [comment]
Same issue here.
